setup/nnt_environment.yml

```diff
@@ -1,100 +1,127 @@
-    - astroid==2.2.5
-    - avryp==0.0.10
-    - configparser==4.0.2
+    - absl-py==0.12.0
+    - anyio==2.2.0
+    - argon2-cffi==20.1.0
+    - astunparse==1.6.3
+    - async-generator==1.10
+    - attrs==20.3.0
+    - babel==2.9.0
+    - backcall==0.2.0
+    - bleach==3.3.0
+    - cachetools==4.2.1
+    - cffi==1.14.5
+    - chardet==4.0.0
+    - colorama==0.4.4
     - cycler==0.10.0
-    - gast==0.2.2
-    - google-pasta==0.1.7
-    - grpcio==1.22.0
-    - h5py==2.9.0
-    - ipykernel==5.1.2
-    - ipython==7.8.0
-    - isort==4.3.21
-    - jedi==0.15.1
-    - joblib==0.13.2
-    - jsonschema==3.0.2
-    - jupyter-client==5.3.3
-    - keras==2.2.4
-    - keras-applications==1.0.8
-    - keras-preprocessing==1.1.0
-    - kiwisolver==1.1.0
-    - lazy-object-proxy==1.4.1
-    - markdown==3.1.1
-    - matplotlib==3.1.1
-    - mccabe==0.6.1
-    - mock==3.0.5
-    - nbconvert==5.6.0
-    - numexpr==2.6.9
-    - numpy==1.16.4
-    - opt-einsum==3.1.0
-    - pandas==0.25.0
-    - parso==0.5.1
-    - pillow==6.2.1
-    - pip==19.2.1
-    - protobuf==3.9.1
-    - pydot==1.4.1
-    - pylint==2.3.1
-    - pyparsing==2.4.2
-    - pyserial==3.4
-    - pytz==2019.2
-    - pyyaml==5.1.2
-    - pyzmq==18.1.0
-    - scikit-learn==0.21.3
-    - scipy==1.3.1
-    - six==1.12.0
-    - sklearn==0.0
-    - tables==3.5.2
-    - tensorboard==2.0.0
-    - tensorflow==2.0.0
-    - tensorflow-estimator==2.0.0
+    - decorator==5.0.7
+    - defusedxml==0.7.1
+    - deprecation==2.1.0
+    - entrypoints==0.3
+    - flatbuffers==1.12
+    - future==0.18.2
+    - gast==0.4.0
+    - google-auth==1.29.0
+    - google-auth-oauthlib==0.4.4
+    - google-pasta==0.2.0
+    - grpcio==1.34.1
+    - h5py==3.1.0
+    - idna==2.10
+    - ipykernel==5.5.3
+    - ipython==7.22.0
+    - ipython-genutils==0.2.0
+    - iso8601==0.1.14
+    - jedi==0.18.0
+    - jinja2==2.11.3
+    - joblib==1.0.1
+    - json5==0.9.5
+    - jsonschema==3.2.0
+    - jupyter-client==6.1.12
+    - jupyter-core==4.7.1
+    - jupyter-packaging==0.9.2
+    - jupyter-server==1.6.4
+    - jupyterlab==3.0.14
+    - jupyterlab-pygments==0.1.2
+    - jupyterlab-server==2.4.0
+    - keras-nightly==2.5.0.dev2021032900
+    - keras-preprocessing==1.1.2
+    - kiwisolver==1.3.1
+    - markdown==3.3.4
+    - markupsafe==1.1.1
+    - matplotlib==3.4.1
+    - mistune==0.8.4
+    - nbclassic==0.2.7
+    - nbclient==0.5.3
+    - nbconvert==6.0.7
+    - nbformat==5.1.3
+    - nest-asyncio==1.5.1
+    - notebook==6.3.0
+    - numpy==1.19.5
+    - oauthlib==3.1.0
+    - opt-einsum==3.3.0
+    - packaging==20.9
+    - pandocfilters==1.4.3
+    - parso==0.8.2
+    - pickleshare==0.7.5
+    - pillow==8.2.0
+    - prometheus-client==0.10.1
+    - prompt-toolkit==3.0.18
+    - protobuf==3.15.8
+    - pyasn1==0.4.8
+    - pyasn1-modules==0.2.8
+    - pycparser==2.20
+    - pygments==2.8.1
+    - pyparsing==2.4.7
+    - pyrsistent==0.17.3
+    - pyserial==3.5
+    - python-dateutil==2.8.1
+    - pytz==2021.1
+    - pywin32==225
```

Comment:
re-adding `pywin32` & `pywinpty` breaks the linux install again. I'm pretty sure you can savely remove those two.